File: PRAKTIKUM_4/soal4.py
while True :
    print("Pilih program \n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n")
    pilihan =int (input("Masukan pilihan : "))
    if pilihan==5:
        print("terimakasih telah menggunaan kalkulator Rahmad :)")
        break
    elif pilihan>=1 and pilihan<=4:
            pertama=int(input('masukan angka pertama : '))
            kedua=int(input('masukan angka kedua : '))
            if pilihan == 1:
                hasil=pertama+kedua
                print(f"Hasil penjumlahan antara {pertama} dengan {kedua} adalah {hasil}\n\n")
            
            elif pilihan==2:
                hasil=pertama-kedua
                print(f"Hasil Pengurangan antara {pertama} dengan {kedua} adalah {hasil}\n\n")
        

            elif pilihan == 3:
                hasil=pertama*kedua
                print(f"Hasil Perkalian antara {pertama} dengan {kedua} adalah {hasil}\n\n")
        
            elif pilihan == 4:
                hasil=pertama/kedua
                print(f"Hasil Pembagian antara {pertama} dengan {kedua} adalah {hasil}\n\n")
    elif pilihan >=6:
            print("inputan anda salah silahkan coba lagi ")

File: PRAKTIKUM_4/soal4.c
#include <stdio.h>
int main(){
    int pilihan;
    float angka1,angka2,hasil;
   
        do {
        printf("Pilih program \n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("masukan pilihan : ");
        scanf("%d",&pilihan);
    if(pilihan>=1&&pilihan<=4){
        printf("masukan nilai pertama : ");
        scanf("%f",&angka1);
        printf("masukan nilai kedua : ");
        scanf("%f",&angka2);

    
        
    if (pilihan == 1){
        hasil=angka1+angka2;
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n",angka1,angka2,hasil);
        
    }else if(pilihan==2){
        hasil=angka1-angka2;
        printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n\n",angka1,angka2,hasil);
        

    }else if(pilihan == 3){
        hasil=angka1*angka2;
        printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n\n",angka1,angka2,hasil);
        
    }else if(pilihan == 4){
        hasil=angka1/angka2;
        printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n\n",angka1,angka2,hasil);
        
    }
    }else if(pilihan>=6){
        printf("Inputan anda salah, silahkan coba lagi\n\n");
    }
    } while (pilihan != 5);
    printf("Terimakasih, telah menggunakan kalkulator Rahmad :^)");
}